### DISCUSSION TITLE
Renovate config not getting created

### DISCUSSION THREAD

**Discussion by haldunanil** [renovatebot/renovate#15700] [Request Help]
I have a legacy repo that essentially acts as a monorepo but has a nonstandard structure in which I'm trying to install renovatebot. The repo shows up in the Renovatebot dashboard app, but no installation PR is ever created and it appears no jobs have ever run. How can I fix this?

**Comment by rarkins** [comment]
It may have got tried previously and marked as disabled, assuming you're talking about the hosted app. Such repos get retried every few weeks or you can tell us the name and we retry it manually

**Reply by haldunanil** [reply]
Yeah it's been "running" for multiple months without any action. The name of the repo is `Appboy/platform`

**Reply by rarkins** [reply]
Actually, the problem is that the installation was suspended by @popsikle `2021-11-01T20:53:35Z`

**Reply by haldunanil** [reply]
it turns out that we turned it off due to a private repository behind a vpn, i ended up moving to self-hosted and it works now. thanks!
